src/topp/MSGFPlusAdapter.cpp

```diff
@@ -722,6 +747,11 @@ class MSGFPlusAdapter :
         vector<ProteinIdentification> protein_ids;
         vector<PeptideIdentification> peptide_ids;
         MzIdentMLFile().load(mzid_temp, protein_ids, peptide_ids);
+        // set the MS-GF+ e-value as new peptide identification score
```

Comment:
"spectral E-value" - to avoid confusion with the other (peptide-level?) E-value that is reported by MS-GF+